src/main/java/thePackmaster/cards/showmanpack/HallOfMirrors.java

```diff
@@ -0,0 +1,31 @@
+package thePackmaster.cards.showmanpack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.green.AfterImage;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.powers.showmanpack.HallOfMirrorsPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class HallOfMirrors extends AbstractShowmanCard {
```

Comment:
So basically you need to pay 3 energy for 1 after image and then wait two turns to get the next one? That seems slow and hard to justify.